src/lint/linter/ArcanistESLintLinter.php

```diff
@@ -147,7 +147,15 @@ protected function parseLinterOutput($path, $err, $stdout, $stderr) {
         $messages = array();
         foreach ($results as $result) {
             $ruleId = idx($result, 'ruleId');
-            $description = idx($result, 'message')."\r\nSee documentation at ".$this->getRuleDocumentationURI($ruleId);
+
+            // Only rules built into eslint are guaranteed to have a rule documentation URI.
+            if (strpos($ruleId, '/') !== FALSE) {
```

Comment:
i totally agree that we should do this but i don't think this is the right place to do it.

i would wait for there to be a conclusion on this:
https://github.com/eslint/eslint/issues/6582

so we can hook into it